dataland-frontend/tests/e2e/fixtures/frameworks/eutaxonomy-financials/EuTaxonomyDataForFinancialsPreparedFixtures.ts

```diff
@@ -139,6 +140,19 @@ function createAllValuesCompany(
   };
   return input;
 }
+/**
+ * Creates a fixture of a company that is an asset management and insurance company
+ * @param input the base fixture to modify
+ * @returns the modified fixture
+ */
+function createCompanyWithBrokenFileReference(
+  input: FixtureData<EuTaxonomyDataForFinancials>
+): FixtureData<EuTaxonomyDataForFinancials> {
+  input = createAllValuesCompany(input);
+  input.companyInformation.companyName = 'TestForIncompleteReferencedReport';
+  input.t.referencedReports = null;
```

Comment:
maybe not null, but instead a random document. (a change here is only necessary if this case is not covered elsewhere. I will update)

Reply:
![image](https://github.com/d-fine/Dataland/assets/92716751/1c0ba55f-9726-4e7f-a5b4-bf40ca6e0009)
applying that change would also cover this I believe

Reply:
only for this preparedfixture. non financial and sfdr are fine as null

Reply:
changed sfdr fixture for additional testing.
removen is collection since eutaxForFinancials doesnt have a collection